specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-02-09/Quotas.json

```diff
@@ -0,0 +1,313 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "2018-02-09",
+        "title": "Compute Admin Client"
+    },
+    "host": "adminmanagement.local.azurestack.external",
+    "schemes": [
+        "https"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Compute.Admin/locations/{location}/quotas/{quotaName}": {
+            "get": {
+                "x-ms-examples": {
+                    "Get an existing Quota.": {
+                        "$ref": "examples/Quotas/Get.json"
+                    }
+                },
+                "tags": [
+                    "Quotas"
+                ],
+                "summary": "Returns the requested quota.",
+                "description": "Get an existing Quota.",
+                "operationId": "Quotas_Get",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "$ref": "../2015-12-01-preview/Compute.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "../2015-12-01-preview/Compute.json#/parameters/LocationParameter"
+                    },
+                    {
+                        "$ref": "Quotas.json#/parameters/QuotaParameter"
+                    },
+                    {
+                        "$ref": "Quotas.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "Quotas.json#/definitions/Quota"
+                        }
+                    },
+                    "404": {
+                        "description": "NOT FOUND"
+                    }
+                }
+            },
+            "put": {
+                "x-ms-examples": {
+                    "Creates or Updates a Quota.": {
+                        "$ref": "examples/Quotas/Create.json"
+                    }
+                },
+                "tags": [
+                    "Quotas"
+                ],
+                "description": "Creates or Updates a Quota.",
+                "summary": "Creates or Updates a Quota.",
```

Comment:
Could you please add better description or summary, to explicitly indicate this is referring to "compute quota" ?

Reply:
This description is on the compute quota object.  The generated documentation will be on the compute quota C# Object and the compute quota PS cmdlet.  Might becoming a little repetitive.